[Workers] examples revision (#3314)
cloudflare/cloudflare-docs: products/workers/src/content/examples/modify-request-property.md

async function handleRequest(request) {
  /**

>    * The best practice is to only assign new properties on the request

Reviewer: ```suggestion
   * The best practice is to only assign new RequestInit properties on the request
```

>    * object (that is, RequestInit props) using either a method or the constructor

Reviewer: ```suggestion
   * on the request object using either a method or the constructor
```
